mlir/test/mlir-miopen-driver/e2e/conv2d_host_validation_yxck_nhwc_nhwk1.mmlir

```diff
@@ -0,0 +1,23 @@
+// COM: yxck/nhwc/nhwk
```

Comment:
Wrong filename: `.mlir` instead of `.mmlir`

Reply:
Oops